resources/views/platform/components/beaten-games-leaderboard/leaderboard-table.blade.php

```diff
@@ -0,0 +1,54 @@
+@props([
+    'isUserOnCurrentPage' => null,
+    'myRankingData' => null, // always null if unauthenticated or no rank
+    'myUsername' => null,
+    'paginator' => null,
+])
+
+<div class="sm:hidden flex flex-col gap-y-1">
+    @foreach ($paginator as $paginatedRow)
+        <x-beaten-games-leaderboard.leaderboard-card-row
+            :myUsername="$myUsername"
+            :paginatedRow="$paginatedRow"
+        />
+    @endforeach
+
+    @if ($myRankingData && !$isUserOnCurrentPage)
+        <x-beaten-games-leaderboard.leaderboard-card-row
+            :isHighlighted="true"
+            :paginatedRow="$myRankingData['userRankingData']"
+            :rank="$myRankingData['userRank']"
+        />
+    @endif
+</div>
+
+<table class="table-highlight hidden sm:table">
+    <thead>
+        <tr class="do-not-highlight">
+            <th width="50px">Rank</th>
+            <th>User</th>
+            <th>Most Recent Game Beaten</th>
+            <th>Beaten When</th>
+            <th class='text-right'>Games Beaten</th>
+        </tr>
+    </thead>
+
+    <tbody>
+        @foreach ($paginator as $paginatedRow)
+            <x-beaten-games-leaderboard.leaderboard-table-row
+                :myUsername="$myUsername"
+                :paginatedRow="$paginatedRow"
+            />
+        @endforeach
+
+        @if ($myRankingData && !$isUserOnCurrentPage)
+            <tr class="do-not-highlight"><td colspan="4">&nbsp;</td></tr>
```

Comment:
I am still seeing this issue and believe it needs to be a colspan of 5 instead of 4, it cuts off slightly early for me still in the latest
![image](https://github.com/RetroAchievements/RAWeb/assets/4047771/0944233e-ad9e-4b9f-a240-a5924c04aba7)

Reply:
I think the reason I'm not seeing it is our browsers have a different opinion of which row to colorize based on the CSS class.

<img width="383" alt="Screenshot 2023-09-12 at 8 30 39 PM" src="https://github.com/RetroAchievements/RAWeb/assets/3984985/748efade-5276-4d54-ba57-c8f99cbfcc85">

Resolved in https://github.com/RetroAchievements/RAWeb/pull/1862/commits/ebe4309effa535db7dc91e682a2130773bbc29ce.

Reply:
Looks like that fixed it, could have also been just landing on an odd or even row based on users I'm not too sure